bodhi/tests/server/test_config.py

```diff
@@ -370,7 +378,7 @@ def test_valid_key(self):
         result = config._validate_fernet_key(key)
 
         self.assertEqual(result, key)
-        self.assertTrue(type(result), str)
+        self.assertTrue(type(result), unicode)
```

Comment:
I think it's actually important that the Fernet key be an str. IIRC, the cryptoengine will raise an Exception if it receives a ```unicode``` object.